drivers/ethernet/Kconfig.sam_gmac

```diff
@@ -12,13 +12,19 @@ menuconfig ETH_SAM_GMAC
 
 if ETH_SAM_GMAC
 
+# Workaround for not being able to have commas in macro arguments
```

Comment:
Happy to drop this patch as I'm only doing it to show how these work.

My main goal is to get the kconfigfunctions.py and edtlib changes merged.